docs/reference/class/MongoDBModelIndexInfo.txt

```diff
@@ -0,0 +1,30 @@
+.. _php-model-index-info:
+
+===============================
+MongoDB\\Model\\IndexInfo Class
```

Comment:
How can I access this page? When I click on this class name, it opens the list of method but doesn't open this page.

<img width="310" alt="image" src="https://github.com/mongodb/mongo-php-library/assets/400034/10646dc3-24ab-417f-8df7-10f5a9389f60">

Reply:
I just updated the snooty.toml file to make the "Enumeration Classes" drawer clickable (similar to the "Result Classes" and "Write Result Classes" drawers). Now you can access this page by going to "Enumeration Classes" -> scrolling down to "MongoDB\\Model\\IndexInfo"